packages/back-end/src/api/openapi/paths/putExperiment.yaml

```diff
@@ -0,0 +1,28 @@
+parameters: []
+tags:
+  - experiments
+summary: Create or update a single experiment
+operationId: putExperiment
+x-codeSamples:
+  - lang: 'cURL'
+    source: |
+      curl -X PUT https://api.growthbook.io/api/v1/experiments \
+        -d '{ id: "test_experiment", name: "test_experiment", "owner": "weijie.ou@discordapp.com" ... }'
```

Comment:
Might want to use an `@example.com` address here since it will be on our public docs.growthbook.io site

Reply:
Fixed.